example/src/components/CallControls.tsx

```diff
@@ -29,12 +26,7 @@ export const CallControls = () => {
   }, [disconnect]);
 
   return (
-    <View
-      style={[
-        styles.iconsContainer,
-        { marginBottom: BOTTOM_MARGIN - bottomOffset },
-      ]}
-    >
+    <View style={[styles.iconsContainer, { marginBottom: BOTTOM_MARGIN }]}>
```

Comment:
What about moving `marginBottom` to the `styles.iconsContainer` and getting rid of `BOTTOM_MARGIN` variable

Reply:
Good catch!